### DISCUSSION TITLE
X and Y coords for track boundaries (inside and outside)

### DISCUSSION THREAD

**Discussion by AreaXimus1** [theOehrly/Fast-F1#564] [Q&A]
get_pos_data() gets me the X Y and Z positional data for a driver around a lap, which is great. 

Is there a way to get the X and Y inside and outside boundaries of the track? I.e. so I could plot the line a driver takes on the tarmac itself?

**Comment by AreaXimus1** [comment]
So I could see something like this

![main-qimg-eee6cf7adfa7b74be7eebef63299a613-lq](https://github.com/theOehrly/Fast-F1/assets/154688263/8865a720-4134-48d8-8004-7bb7c6342e9f)

**Comment by theOehrly** [comment]
Hi, what you want to do is sadly not possible. The coordinates aren't real coordinates like GPS.

See this answer here for more details: https://github.com/theOehrly/Fast-F1/discussions/116#discussioncomment-2379410

**Reply by AreaXimus1** [reply]
Ah damn. Thanks for your response!
